test/e2e/reconciler/cluster/controller_test.go

```diff
@@ -67,28 +67,29 @@ func TestClusterController(t *testing.T) {
 		{
 			name: "create an object, expect spec and status to sync to sink, then delete",
 			work: func(ctx context.Context, t *testing.T, servers map[string]runningServer, syncerFixture *framework.StartedSyncerFixture) {
+				kcpClient, err := kcpclientset.NewForConfig(syncerFixture.SyncerConfig.UpstreamConfig)
+				require.NoError(t, err)
+
+				syncTarget, err := kcpClient.WorkloadV1alpha1().SyncTargets().Get(ctx,
+					syncerFixture.SyncerConfig.SyncTargetName,
+					metav1.GetOptions{},
+				)
+				require.NoError(t, err)
+				syncTargetKey := workloadv1alpha1.ToSyncTargetKey(syncTarget.GetName(), logicalcluster.From(syncTarget))
```

Comment:
why this change? Just take `syncerFixture.SyncerConfig.SyncTargetName` and `syncerFixture.SyncerConfig.SyncTargetWorkspace` ?

Reply:
I get the syncTarget for the UID (line 91) and then reuse the name/workspace here, but I can use the ones from fixture.